app/views/streams/main_stream.html.haml

```diff
@@ -20,38 +20,144 @@
           .pull-right
             = link_to '&times;'.html_safe, getting_started_completed_path, id: "gs-skip-x", class: "close"
```

Comment:
This block looks like it could need some love. I'd say we could use `.container-fluid` instead of `.container`, remove that `.offset1` which wasn't working at all and instead use `.col-md-9` and `.col-md-3` or something similar.